lib/ex_cell/ex_cell.ex

```diff
@@ -12,4 +13,36 @@ defmodule ExCell do
     |> Application.get_env(__MODULE__, [])
     |> Keyword.get(keyword, fallback)
   end
+
+  def relative_name(module, namespace) do
+    parts = case namespace do
+      nil -> Module.split(module)
+      _ -> ExCell.module_relative_to(module, namespace)
+    end
+
+    Enum.join(parts, "-")
+  end
+
+  def class_name(name, classes) do
+    [name, classes]
+    |> List.flatten
```

Comment:
`List.flatten()` 👮

Reply:
We actually haven't done that in this codebase yet.